### ISSUE TITLE
PickList: add flag to disable dragdrop support

### ISSUE THREAD

**Issue by BalusC** [primefaces/primefaces#13131]
### Description

Would be nice if PrimeFaces [`<p:pickList>`](https://www.primefaces.org/showcase/ui/data/pickList.xhtml) allows disabling dragdrop support (among others to improve user experience on touchscreen devices).

Note: this is already implemented in PrimeNG [`<p-picklist>`](https://primeng.org/picklist).

### Describe the solution you would like

Add a new boolean attribute like `dragdrop` in PrimeNG one.

**Comment by tandraschko** [comment]
alternative: disable it always on touch devices per default
AFAIR we already have somethign for other components

**Comment by melloware** [comment]
Yep we do something similar where is touch enabled is detected we change the behavior. I will addd this.
